lib/killbill_client/models/credit.rb

```diff
@@ -18,6 +18,14 @@ def create(auto_commit = false, user = nil, reason = nil, comment = nil, options
         created_credit.refresh(options)
       end
 
+      def retrieve_credit(credit_id, options = {})
```

Comment:
For consistency with other methods, let's rename it to `find_by_id`.

Also, it should be a class method (none of the instance attributes are in use).

Reply:
Done!